extensions/ql-vscode/src/stories/model-alerts/ModelAlerts.stories.tsx

```diff
@@ -3,7 +3,7 @@ import type { Meta, StoryFn } from "@storybook/react";
 import { ModelAlerts as ModelAlertsComponent } from "../../view/model-alerts/ModelAlerts";
 
 export default {
-  title: "CodeQL Model Alerts/CodeQL Model Alerts",
+  title: "Model Alerts/Model Alerts",
   component: ModelAlertsComponent,
 } as Meta<typeof ModelAlertsComponent>;
 
```

Comment:
I can't get anything to appear in the main "model alerts" story (just the new "model alerts header" story) 🤔 

I could delete it, but I'm probably just doing something stupid to prevent things from working 😅